drivers/include/sx1280.h

```diff
@@ -0,0 +1,1095 @@
+/*!
+ * @file      sx1280.h
+ *
+ * @brief     SX1280 radio driver definition
+ *
+ * Revised BSD License
+ * Copyright Semtech Corporation 2020. All rights reserved.
```

Comment:
All files in the driver have the Semtech copyright. They seem to just be copy-pasted into RIOT. I think only `sx1280_hal.c` contains the adaption code to RIOT's API.

Is there an original repo from where this code was taken ? If yes, I strongly recommend to use the package mechanism, like it was done for the sx126x driver.

Reply:
The repo with vendor code seems to be here : https://github.com/lorabasics/lorabasicsmodem https://github.com/lorabasics/lorabasicsmodem/tree/master/sx1280_driver

Reply:
Thanks! After a quick look, I think it's possible to use the sx1280_driver part in a package, similar to what was done for the sx126x driver. And as @fjmolinas pointed-out, it would be nice to provide a netdev adaptation layer to the driver. Not sure how that would work with LoRaWAN (is it possible ?).